pages/Favorites/subscriptions.js

```diff
@@ -14,20 +14,21 @@ export default function favorites(subscribe) {
     dispatch(setTitle('titles.favorites'));
   });
 
-  subscribe(favoritesWillRemoveItem$, ({ dispatch, action, getState }) => {
+  subscribe(favoritesWillRemoveItem$, ({
+    action, dispatch, events, getState,
+  }) => {
     if (getCurrentRoute(getState()).pattern !== FAVORITES_PATH) {
```

Comment:
You should change this to use the router helper function instead.

Reply:
Changed